src/rgw/rgw_kafka.cc

```diff
@@ -632,6 +634,20 @@ class Manager {
       ldout(cct, 1) << "Kafka connect: user/password are only allowed over secure connection" << dendl;
       return false;
     }
+    
+    if (brokers.has_value()) {
+      std::string kafka_brokers = brokers.get();
```

Comment:
is librdkafka going to error if the same broker is going to be twice in the list?
if not, then you can probably avoid the split and just call `std::string::append()`.
note that identifying duplicate broker, can be a challange if you take into account ports and default ports

either way, the names are confusing, could you please rename `broker` to `brokers` or `broker_list` ?

Reply:
@thuvh please look at the above

Reply:
I used append to update broker list, and test with test_bn, I think librdkafka can handle case duplicate host:port